os/arch/arm/src/armv7-r/arm_assert.c

```diff
@@ -972,5 +980,19 @@ void up_assert(const uint8_t *filename, int lineno)
 	(void)boardctl(BOARDIOC_RESET, 0);
 #endif
 
+#ifdef CONFIG_FAULT_MGR
+	fault_queue = mq_open("fault_queue", O_WRONLY | O_CREAT,  0666, 0);
```

Comment:
Please define queue name, "fault_queue" in header

Reply:
Defined